src/main/java/seedu/address/ui/TabBar.java

```diff
@@ -112,7 +102,7 @@ public void setTab(EntityType entityType) {
             tabBar.getSelectionModel().select(propertyTab);
         }
         if (entityType == EntityType.MEETING) {
-            tabBar.getSelectionModel().select(calenderTab);
+            // tabBar.getSelectionModel().select(calenderTab);
```

Comment:
wait why is their a commented out bit so if the entity type is Meeting it does nth?